packages/twenty-server/src/modules/messaging/services/gmail-error-handling/gmail-error-handling.service.ts

```diff
@@ -164,6 +164,11 @@ export class GmailErrorHandlingService {
       messageChannel.connectedAccountId,
       workspaceId,
     );
+
+    await this.connectedAccountRepository.updateAccessTokenNeedsRefresh(
+      messageChannel.connectedAccountId,
+      workspaceId,
+    );
```

Comment:
Probably not the correct place to put this into, but couldnt find a better place so its not repeated in every single sync job. Let me know if you have any other idea.